app/background/api/cosmeticFilterAPI.ts

```diff
@@ -0,0 +1,41 @@
+// import { Tab } from '../../types/state/shieldsPannelState'
+// import { getShieldSettingsForTabData, getTabData } from './shieldsAPI'
+
+export const addSiteCosmeticFilter = async (origin: string, cssfilter: string) => {
+  chrome.storage.local.get('list', (result = {}) => {
+    let l = Object.assign({}, result.list)
```

Comment:
nit: can we have a more descriptive variable name here?

Reply:
are you referring to `list`, `l`, `result`, or any combination of those 3?

Reply:
ya I mean the `l`

Reply:
fix'd